File: Data Structures & Algorithms/palindromic-substrings/submission-1.cpp
class Solution {
public:
    int countSubstrings(string s) {
        int n =s.size() , res=n;
        for (int i=0;i<n;i++){
            int l=i-1,r=i+1;
            while (i>=0 && r<n && s[l]==s[r]){
                l--;
                r++;
                res+=1;
            }
            l=i,r=i+1;
            while (i>=0 && r<n && s[l]==s[r]){
                l--;
                r++;
                res+=1;
            }
        }
        return res;
    }  
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        res = []
        seen = {}
        for i in range(len(nums)):
            goal= target-nums[i]
            if goal in seen.keys():
                res.append(seen.get(goal))
                res.append(i)
                return res
            seen[nums[i]]=i
        return 
        